commercial/app/model/merchandise/Merchandise.scala

```diff
@@ -203,35 +210,27 @@ object Masterclass {
       }
     }
 
-    def extractFirstParagraph(html: String) = {
-      val firstParagraph: Option[Element] = Some(doc.select("p").first())
-      firstParagraph match {
-        case Some(p) => p.text
-        case _ => ""
-      }
-    }
-
     extractGuardianUrl map { extractedUrl =>
 
       new Masterclass(
         id = event.id,
         name = event.name,
-        startDate = event.startDate,
+        startDate = event.start_date,
```

Comment:
I think if you're going to convert something into a Scala case class you should probably follow the Scala conventions.  Otherwise you could do [coast-to-coast json](https://www.playframework.com/documentation/2.5.x/ScalaJsonTransformers) instead.  But if there's a good reason for abandoning the convention you should put a comment on the field to avoid someone refactoring it later.